source/_speakers.html.haml

```diff
@@ -21,7 +21,8 @@
     %a{:name=>"joe"}
       %h3 Bermon Painter
     .bio
-      Bermon Painter is the UX Design Lead for Hendrick Automotive Group, where he works with an amazing team that crafts nifty experiences and web applications with Ruby. He's also heavily involved in the local tech community in Charlotte and organizes the Charlotte User Experience Designer, Charlotte Front-End Developers and Charlotte Grok groups. In his free time he works the folks at @teamsassdesign on the redesign of the Sass website.
+      Bermon is the organizer of various community groups for user experience designers and front-end developers, and the organizer of Blend Conference, a 3-day multi-track event for user experience strategists, designers and developers. He also leads the user experience team for Cardinal Solutions' Charlotte office where he consults with large enterprise clients on interesting problems across user experience, design and front-end development. 
+
```

Comment:
Argh I totally merged this before seeing your comment :(

@bpainter – did you want to update your bio to add in the Team Sass Design stuff or should I?